lib/mixins/listen-on-root.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Issue #569: collapse::toggle::state triggered too many times
+ * @link https://github.com/bootstrap-vue/bootstrap-vue/issues/569
+ */
+export default {
+    data() {
```

Comment:
P.S. There's really an art to selecting the line number to comment on. This giant box can really get in the way!